sqlalchemy-netezza/nzalchemy/__init__.py

```diff
@@ -84,6 +84,7 @@
 #__version__ = "11.0.0"
 _registry.register(
     "netezza.pyodbc", "nzalchemy.pyodbc", "NetezzaDialect_pyodbc"
+    #"netezza.nzpy", "nzalchemy.nzpy", "NetezzaDialect_nzpy"
```

Comment:
please correct this. pyodbc will no longer be used.

Reply:
As discussed, we can keep both pyodbc and nzpy in repository. User can decide which one to use.